pubspec.yaml

```
@@ -5,7 +5,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 0.0.1-alpha.3+1
 
 environment:
-  sdk: ">=2.12.3 <3.0.0"
+  sdk: ">=3.0.0 <4.0.0"
```

```suggestion
  sdk: ">=3.0.0 <4.0.0"
```

we not bump till >2.15? will some requirements stop us from using dart 2.x?

---

I tried running the app after bumping to flutter 3. It works fine but we can downgrade the sdk to support larger dependency.

---

I think `">=2.12.3 <4.0.0"` woul be nice

I accepted a similar PR yesterday in https://github.com/zino-hofmann/graphql-flutter/pull/1328

---

Sure!